### ISSUE TITLE
It's not working with Android

### ISSUE THREAD

**Issue by mohitrajput987** [schlan/java-censor#6]
Hi,

I'm using this library with Android to censor my module project but when I created AAR file using assembleRelease task and used .aar file in a project, I can see full code in my library classes.
I have done following things in my module level build.gradle file-

1. Added apply plugin: 'com.sebchlan.javacensor' at the top of the file
2. Added following buildscript section just below the apply plugin - 
buildscript {
    repositories {
        jcenter()
    }

    dependencies {
        classpath group: 'com.sebchlan.javacensor', name: 'java-censor', version: '1.1.0'
    }
}
3. Added after evaluate section in android section.

What am I doing wrong here?

**Comment by mohitrajput987** [comment]
Is there any solution for this?

**Comment by schlan** [comment]
Hey, sorry for the late response. If it's still an issue, can you post your full build.gradle ?

**Comment by mohitrajput987** [comment]
Yes it's still an issue. I will make the project again and send you.
